File: Assignment13/program3.c
/////////////////////////////////////////////////////////////////// 
//
// File name : program3.c
// Description : Pattern
// input : iRow = 3  iCol = 5
// Output : 
/*
    A   A   A   A   A
    B   B   B   B   B
    C   C   C   C   C
*/
// Author : Adinath Santosh Pawar
// Date : 27/5/25
// 
///////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern(int iRow,int iCol)
{
    int j = 0;
    int i = 0;
    char ch = 'A';

    for(i = 1; i <= iRow; i++,ch++)
    {    
        for(j = 1; j <= iCol; j++)
        {
           printf("%c\t",ch);
        }
        printf("\n");
    }

}

int main()
{
    int iValue1 = 0, iValue2 = 0;

    printf("enter number of rows \n");
    scanf("%d",&iValue1);

    printf("enter number of columns \n");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);
    return 0;
}

File: Assignment45/program3.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program3.java
// Description : reverse content of each column of matrix
// Author : Adinath Santosh Pawar
// Date : 2/8/25
// 
///////////////////////////////////////////////////////////////////

class Matrix
{
    private int Arr[][];

    public Matrix(int A, int B)
    {
        Arr = new int[A][B];
    }
        
    public void Accept()
    {
        System.out.println("Enter the elements of matrix ");

        Scanner sobj = new Scanner(System.in);

        int i = 0, j = 0;
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                Arr[i][j] = sobj.nextInt();
            }
        }
    }

    public void ReverseColumn(int iRow,int iCol)
    {
        int i = 0, j = 0;
        for(i = 0; i < Arr.length / 2; i++)
        {
            for(j = 0; j < (Arr[i].length); j++)
            {
                int temp = Arr[i][j];
                Arr[i][j] = Arr[iRow-i-1][j];
                Arr[iRow-i-1][j] = temp;
            }
        }
    }

    public void Display()
    {
        System.out.println("The elements of matrix are ");

        int i = 0, j = 0;
        for(i = 0; i < Arr.length; i++)
        {
            for(j = 0; j < Arr[i].length; j++)
            {
                System.out.print(Arr[i][j] + " ");
            }
            System.out.println();
        }
    }

}
class program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iRet = 0;

        System.out.println("Enter number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter number of columns :");
        int iCol = sobj.nextInt();

        Matrix mobj = new Matrix(iRow,iCol);

        mobj.Accept();
        mobj.ReverseColumn(iRow,iCol);
        mobj.Display();
    }
}